src/python/bot/tasks/commands.py

```diff
@@ -84,7 +84,9 @@ def cleanup_task_state():
   environment.reset_current_memory_tool_options()
 
   # Clear exceptions.
-  sys.exc_clear()
+  if sys.version_info.major == 2:
+    # TODO(ochang): Remove this once migrated to Python 3.
+    sys.exc_clear()
```

Comment:
i think we use this in 3 places, feel free to create a helper in utils.py and use it

src/python/bot/startup/heartbeat.py:        sys.exc_clear()
src/python/bot/startup/run_bot.py:    sys.exc_clear()
src/python/bot/tasks/commands.py:  sys.exc_clear()

Reply:
Done!